modules/influxdb/influxdb.go

```diff
@@ -120,6 +122,142 @@ func WithConfigFile(configFile string) testcontainers.CustomizeRequestOption {
 	}
 }
 
+// withV2 configures the influxdb container to be compatible with InfluxDB v2
+func withV2(req *testcontainers.GenericContainerRequest, org, bucket string) error {
+	if org == "" {
+		return errors.New("organization name is required")
+	}
+
+	if bucket == "" {
+		return errors.New("bucket name is required")
+	}
+
+	req.Env["DOCKER_INFLUXDB_INIT_ORG"] = org
+	req.Env["DOCKER_INFLUXDB_INIT_BUCKET"] = bucket
+	req.Env["DOCKER_INFLUXDB_INIT_MODE"] = "setup" // Always setup, we wont be migrating from v1 to v2
+	return nil
+}
+
+// WithV2 configures the influxdb container to be compatible with InfluxDB v2
+func WithV2(org, bucket string) testcontainers.CustomizeRequestOption {
+	return func(req *testcontainers.GenericContainerRequest) error {
+		err := withV2(req, org, bucket)
+		if err != nil {
+			return err
+		}
+
+		return nil
+	}
+}
+
+const dockerSecretPath = "/run/secrets"
+
+func secretsPath(path string) string {
+	return dockerSecretPath + "/" + path
+}
```

Comment:
question: this means that a file must be created in the container, right? How are we setting it? I see the env var initialisation but I'm not able yet to check where the file is copied into the container. Is that assumption correct?

Reply:
Never mind, I realised this is the Docker secrets path. Candidate to live in the core for a follow-up